dashboard/test/ui/features/teacher_tools/level_types/bubble_choice.feature

```diff
@@ -25,15 +24,6 @@ Feature: BubbleChoice
     # View student's BubbleChoice progress as a teacher
     When I sign in as "Teacher_Alice"
 
-    # View progress from script overview page
-    Given I am on "http://studio.code.org/s/allthethings"
```

Comment:
If we want to maintain this test functionality, we should probably switch from the teacher panel to the `viewing progress for` dropdown that is now part of the page. 

Do you think it's worth it to change or should we just remove?